Auth-MSSC/src/main/java/com/clinics/auth/bootstrap/BootStrapUser.java

```diff
@@ -30,54 +30,55 @@ public void run(String... args) throws Exception {
 	public void init(){
 		User userPatient1 = User
 				.builder()
-				.uuid(UUID.randomUUID())
+				.userUUID(UUID.randomUUID())
 				.email("jan@jan.pl")
 				.password(passwordEncoder.encode("12345"))
 				.role(Role.PATIENT)
 				.build();
 		User userPatient2 = User
 				.builder()
-				.uuid(UUID.randomUUID())
+				.userUUID(UUID.randomUUID())
 				.email("adam@adam.pl")
 				.password(passwordEncoder.encode("12345"))
 				.role(Role.PATIENT)
 				.build();
 		User userDoctor1 = User
 				.builder()
-				.uuid(UUID.fromString("03f0f891-b243-4547-803b-605f72b11be9"))
+				.userUUID(UUID.fromString("03f0f891-b243-4547-803b-605f72b11be9"))
 				.email("ola@ola.pl")
 				.password(passwordEncoder.encode("12345"))
 				.role(Role.DOCTOR)
 				.isEnable(true)
```

Comment:
The name of this variable `isEnable` is unfortunate. Let's refactor it to `isActive`, or maybe even only `active`

Reply:
UserService provided method isEnabled so I propose to change this variable into isEnabled